cmd/kubeadm/app/phases/upgrade/postupgrade.go

```diff
@@ -281,3 +289,53 @@ func GetKubeletDir(dryRun bool) (string, error) {
 	}
 	return kubeadmconstants.KubeletRunDirectory, nil
 }
+
+func UpdateKubeletLocalMode(cfg *kubeadmapi.InitConfiguration, dryRun bool) error {
+	// TODO(chrischdi): how to get the correct dir? kubeadm init has a flag to change the location
+	dir := kubeadmconstants.KubernetesDir
```

Comment:
@neolit123 : should I add a flag to the relevant `kubeadm upgrade` subcommands similar to `kubeadm init`'s 
`--kubeconfig-dir` ([source](https://github.com/kubernetes/kubernetes/blob/e51b55b45110d7276cbe65ce789c37e2515880a4/cmd/kubeadm/app/cmd/options/generic.go#L39))?

Reply:
no i don't think so.
the flag is only available for `kubeadm init phase kubeconfig all` or other subphases of kubeconfig. it's a generative phase, but the supported location remains only /etc/kubernetes/kubelet.conf for init/join/upgrade